docs/src/build.md

```diff
@@ -111,11 +111,14 @@ sudo apt install ninja-build
 
 ### Ubuntu 20.04
 
-On Ubuntu 20.04, we need to update the version because 3.20 is the minimum required for this project
+On Ubuntu 20.04 we need to instal clang and to update the version of cmake because 3.20 is the minimum required for this project.
 
 ```bash
 sudo apt-get update
 sudo apt-get upgrade
+
+sudo apt-get install clang
```

Comment:
Is this needed? I thought ubuntu 20.04 ships with clang.

Reply:
I think it doesn't :(

Reply:
I think it doesn't :(